src/selectors/entities/users.js

```diff
@@ -177,14 +177,18 @@ export const getProfileSetNotInCurrentTeam = createSelector(
     }
 );
 
-function sortAndInjectProfiles(profiles, profileSet) {
+function sortAndInjectProfiles(profiles, profileSet, skipInactive = false) {
```

Comment:
what is this used for? should we do anything in RN for this?

Reply:
To skip inactive users, the default is the previous behaviour so you shouldn't need to change anything

Reply:
Hmm, I wonder if we should turn this around as I don't really see any reasons to show inactive users other than in the system console

Reply:
Maybe, but that would need to be done because that would require more changes in the webapp that I don't have time to do now